### PR TITLE
feat: connect create-page api [WIP]

### PR DESCRIPTION
**Pull request by minseolee**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

SSIA
Commits are for development convinience
Sorry for late providing

### CODE REVIEW

File: src/services/dashboards/dashboard-create/DashboardCreatePage.vue [hunk 1]
```diff
@@ -84,12 +86,16 @@ export default {
...
+                viewers: state.dashboardViewerType,
             };
-            console.log(dashboardCreateParams);
+
+            if (state.dashboardScope === DASHBOARD_SCOPE.PROJECT) {
+                SpaceConnector.clientV2.dashboard.projectDashboard.create({
+                    ...dashboardCreateParams,
+                    project_id: dashboardProject.value?.id ?? '',
+                });
+            } else SpaceConnector.clientV2.dashboard.domainDashboard.create(dashboardCreateParams);
```
Comment (sulmoJ) [inline src/services/dashboards/dashboard-create/DashboardCreatePage.vue:99]: I think it'd be good if there was try catch!

**Review by yuda110 — APPROVED** [review]
LGTM!
I left a comment!

**Review by sulmoJ — APPROVED** [review]
LGTM!!
I left a comment!

File: src/services/dashboards/dashboard-create/DashboardCreatePage.vue (hunk 1 above)
Comment (minseolee) [reply src/services/dashboards/dashboard-create/DashboardCreatePage.vue:99]: I forgot it .. Thanks a lot !

**Pull request merged**